cmake/externals/opencv.cmake

```diff
@@ -93,12 +93,15 @@ FetchContent_Declare(
         -DBUILD_SHARED_LIBS:BOOL=FALSE
         -DCMAKE_INSTALL_PREFIX:PATH=${CMAKE_CURRENT_BINARY_DIR}/opencv
         -DCV_TRACE:BOOL=FALSE
-        PATCH_COMMAND  git apply --whitespace=fix ${CMAKE_CURRENT_SOURCE_DIR}/cmake/externals/opencv-no-rtti.patch
+        PATCH_COMMAND git checkout . && git apply --whitespace=fix --reject --allow-empty ${CMAKE_CURRENT_SOURCE_DIR}/cmake/externals/opencv-no-rtti.patch

```

Comment:
Try this: https://github.com/microsoft/onnxruntime/blob/master/cmake/external/abseil-cpp.cmake

I suggest avoid using git apply, use patch directly.

Reply:
patch command line is only available on Unix-like systems, and git apply command is same among all platforms. so it looks like it is better.